src/components/ProcessIndicator.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Grid, Row, Col } from 'react-flexbox-grid';
+import IconButton from 'material-ui/IconButton';
+import LinearProgress from 'material-ui/LinearProgress';
+
+import styles from './ProcessIndicator.scss';
+import common from '../styles/common.scss';
+
+// TODO: this component should be reviewed with Ola for approval.
+// For now its more of placeholder to continue most urgent work
+
+const ProcessIndicatorComponent = ({ stepsTotal, stepNumber, stepName }) =>
+  (<div className={styles.processIndicator}>
+    <div className={common.layoutWidthLimiter}>
+      <Grid fluid>
+        <Row>
+          <Col lg={1} className={`${styles.column} ${styles.back}`}>
+            {/* TODO: consult with designer exact sizes and positions of back arrow*/}
+            <IconButton
+              iconClassName="material-icons"
+              iconStyle={{ color: '#E8EFE0' }}
```

Comment:
Can this go in the CSS file somehow? Would be nice to have all the colours in one place.